kclvm/sema/src/resolver/type_erasure.rs

```diff
@@ -0,0 +1,30 @@
+use kclvm_ast::ast;
+use kclvm_ast::walker::MutSelfMutWalker;
+
+#[derive(Default)]
+struct TypeErasureTransformer {}
+
+impl<'ctx> MutSelfMutWalker<'ctx> for TypeErasureTransformer {
+    fn walk_schema_attr(&mut self, schema_attr: &'ctx mut ast::SchemaAttr) {
```

Comment:
There should be more than just these two places here, such as index signature type, lambda, schema parameter type, etc. You can search the AST definition as a whole...

Reply:
fixed.